packages/core/content-type-builder/admin/src/components/FormModal/forms/index.js

```diff
@@ -186,11 +187,16 @@ const forms = {
           })
         : pluralNames;
 
+      const collectionNames = Object.values(contentTypes).map((contentType) => {
```

Comment:
The comment that I mentioned in the other comment would be appreciated here I think